modules/microshift-firewall-opt-settings.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+
+[id="microshift-firewall-optional-settings_{context}"]
+== Optional port settings
+
+.Procedure
+
+. To add customized ports to your firewall configuration, use the following command syntax:
++
+[source,terminal]
+----
+$ sudo firewall-cmd --permanent --zone=public --add-port=<port number>/<port protocol>
+----
++
+.Optional ports
+[option="header"]
+|===
+|Port(s)|Protocol(s)|Description
+
+|80
+|TCP
+|HTTP port used to serve applications through the {ocp} router.
+
+|443
+|TCP
+|HTTPS port used to serve applications through the {ocp} router.
+
+|5353
+|UDP
+|mDNS service to respond for {ocp} route mDNS hosts.
+
+|30000-32767
+|TCP
+|Port range reserved for NodePort services; can be used to expose applications on the LAN.
+
+|30000-32767
+|UDP
+|Port range reserved for NodePort services; can be used to expose applications on the LAN.
+
+|6443
+|TCP
+|HTTPS API port for the {product-title} API.
+|===
+
+The following are examples of commands used when requiring external access through the firewall to services running on {product-title}, such as port 6443 for the API server, ports 80 and 443 for applications exposed through the router, etc.
+
+.Example commands:
```

Comment:
You might check the repo for this, but in MOST cases we don't use a colon after commands here. There are some in the repository but I think when we use ".Example x", we don't use a colon. eg, ".Example output"